consolidated walkthrough (#383)
trinsic-id/sdk: docs/dotnet/index.md

# The Trinsic C# / .NET SDK  
The Trinsic C# / .NET SDK makes it easy to interact with the Trinsic API from your .NET application. The most recent version of the library can be found on NuGet. The Trinsic SDK supports .NET applications written in C#, VB.NET, and F# that utilize any supported version of .NET Core. You can also find the SDKs source on [Github](https://github.com/trinsic-id/sdk/dotnet).

!!! note "Supported runtimes"

>     Xamarin targets for iOS and Android are fully supported using the same package dependencies. Support for Blazor in Web Assembly will be available in a future release.

Reviewer: This is not needed, all .NET runtimes are now supported.

> 
> !!! note "Supported runtimes"
>     Xamarin targets for iOS and Android are fully supported using the same package dependencies. Support for Blazor in Web Assembly will be available in a future release.

Reviewer 2: ```suggestion

!!! note "Supported runtimes"
    .NET targets for iOS, Android, and Blazor are fully supported using the same package dependencies via .NET 6.
```

## Installation in a new project
Add the required dependencies from [Nuget.org :material-open-in-new:](https://www.nuget.org/packages/Trinsic)

=== "Package Manager"
    ```
    PM> Install-Package Trinsic

>     PM> Install-Package Okapi.Net 

Reviewer: This is not required, Okapi.Net transitively brings everything it needs now. Check dotnet/README

>     PM> Install-Package Okapi.Net 

Reviewer 2: ```suggestion
```

    ```
=== ".NET CLI"
    ```bash
    dotnet add package Trinsic --prerelease
    dotnet add package Okapi.Net --prerelease
    ```
=== "PackageReference"
    ```
    <PackageReference Include="Trinsic" />
    <PackageReference Include="Okapi.Net" />
    ```

!!! note ""

>     The package `Okapi.Net` is already a dependency of `Trinsic` package, but we must add explicitly to bring in static library dependencies.

Reviewer 2: we still have to install the okapi native libraries, don't we?

> !!! note ""
>     The package `Okapi.Net` is already a dependency of `Trinsic` package, but we must add explicitly to bring in static library dependencies.

Reviewer 2: ```suggestion
```

    This will not be required in future version and only package `Trinsic` can be used.

## Configuration

```csharp
using Trinsic;

// Set the server address

> var service = new WalletService();

Reviewer: I think wallet service ctor needs an account profile now.

Reviewer 2: it does

> var service = new WalletService();

Reviewer 2: ```suggestion
// WalletService requires an account profile
var accountService = new AccountService();
var accountProfile = await accountService.signIn();
var service = new WalletService(accountProfile);
```
